File: crates/zyntax_python/pressure/strings.py
# A string built, measured and dropped every step; the accumulator is
# an int so nothing keeps them.
import sys

def main(n: int) -> int:
    total = 0
    for i in range(n):
        s = "step " + str(i)
        t = s + "!"
        total += len(t)
    return total

print(main(int(sys.argv[1])))

File: crates/zyntax_lua/pressure/strings.lua
-- A string built, measured and dropped every step; the accumulator is
-- an integer so nothing keeps them.
local function main(n)
  local total = 0
  for i = 1, n do
    local s = "step " .. i
    local t = s .. "!"
    total = total + #t
  end
  return total
end

print(main(tonumber(arg[1])))
